community-membership.md

```diff
@@ -220,6 +219,26 @@ The following apply to the subproject for which one would be an owner.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
 
+### Becoming a Maintainer
+
+Unless stated otherwise in a SIG charter ratified by the Technical Committee,
+a new maintainer is elected by unanimous vote of the existing maintainers
+of the SIG. After a successful vote, a pull request should be made adding
+the new maintainer to the list of existing maintainers. Before the pull request
+is merged, all existing maintainers and the newly elected maintainer should
+approve the pull request to show their support for the public record. The
+nominee is considered a maintainer after the pull request is merged.
+
+If you feel like you meet the requirements above and are willing to take on the
+additional responsibilities and privileges of being a maintainer, it is
+recommended that you approach an existing maintainer about sponsoring your bid
+to become a maintainer. After you and your sponsor have discussed the role
+and its additional requirements and responsibilities, they may approach the other
```

Comment:
Should we list the common expectations connected to this role? Like:

- performing releases
- answering users questions
- fixing bugs across the entire code base
- mentoring other prospective maintainers

Reply:
Isn't that listed above in requrements/responsibilities/privileges sections?

Reply:
Sorry about that, it indeed is!